### DISCUSSION TITLE
Someone please help

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [KadeArchive/Kade-Engine#2126] [Q&A] [answered]
When I added my character into all the files, I did "lime test windows" but I got an error saying

source/Character.hx:384: characters 5-14 : Missing ;

How do I fix this, because I went to line 384, surely enough, it was my character, but I went to characters 5-14 and somewhat couldn't find what was wrong
Please help

-Struggling Mod Developer

**Comment by SanicBTW** [comment, marked as answer]
It's telling you right there "missing ;" you have to add the ; to the end of the line or the character

**Reply by THEkatinamicrowave** [reply]
thank you! I'll try that

-SMD

**Reply by THEkatinamicrowave** [reply]
source/Character.hx:384: characters 5-62 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool
source/Character.hx:385: characters 5-61 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool
source/Character.hx:386: characters 5-65 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool
source/Character.hx:387: characters 5-65 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool
source/Character.hx:388: characters 5-67 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool
source/Character.hx:389: characters 5-70 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool
source/Character.hx:390: characters 5-74 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool
source/Character.hx:391: characters 5-74 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool
source/Character.hx:392: characters 5-76 : Not enough arguments, expected Prefix:String, FrameRate:Int, Looped:Bool, FlipX:Bool, FlipY:Bool

What is this and how do I fix it

**Reply by THEkatinamicrowave** [reply]
Also my code for this is 

frames = Paths.getSparrowAtlas('characters/JeffNorm');
animation.addByPrefix('idle, Jeff idle dance, 24, false');
animation.addByPrefix('singUP, Jeff NOTE UP, 24, false');
animation.addByPrefix('singDOWN, Jeff NOTE DOWN, 24, false');
animation.addByPrefix('singLEFT, Jeff NOTE LEFT, 24, false');
animation.addByPrefix('singRIGHT, Jeff NOTE RIGHT, 24, false');
animation.addByPrefix('singUPmiss, Jeff NOTE UP MISS, 24, false');
animation.addByPrefix('singDOWNmiss, Jeff NOTE DOWN MISS, 24, false');
animation.addByPrefix('singLEFTmiss, Jeff NOTE LEFT MISS, 24, false');
animation.addByPrefix('singRIGHTmiss, Jeff NOTE RIGHT MISS, 24, false');

If it helps, I'm posting it

**Reply by SanicBTW** [reply]
Wait never mind, it says not enough arguments, you have to add another bool that should be false because it's flipY try adding `, false` to all your animation prefixes

**Reply by THEkatinamicrowave** [reply]
Thank you that seems to have done it

**Reply by THEkatinamicrowave** [reply]
What'w your discord so I can come to you with stuff quickly

**Reply by SanicBTW** [reply]
sanic#9704

**Reply by THEkatinamicrowave** [reply]
thank
